File: Level2/예상 대진표.cpp
#include <iostream>

using namespace std;

int solution(int n, int a, int b)
{
    int answer = 0;
    
    while(1)
    {
        a = (a / 2) + (a % 2);
        b = (b / 2) + (b % 2);
        answer++;
        
        if(a == b)
            return answer;
    }
}

File: Level2/예상 대진표.js
function solution(n,a,b)
{
    let answer = 0;
    
    while(1){
        a = parseInt((a / 2) + (a % 2));
        b = parseInt((b / 2) + (b % 2));
        answer++;
        
        if(a === b){
            return answer;
        }
    }
}